examples/demo/src/reviews/reviewSaga.js

```diff
@@ -1,23 +1,9 @@
 import { all, put, takeEvery } from 'redux-saga/effects';
-import { push } from 'react-router-redux';
 import { showNotification } from 'react-admin';
-import {
-    REVIEW_APPROVE_SUCCESS,
-    REVIEW_APPROVE_FAILURE,
-    REVIEW_REJECT_SUCCESS,
-    REVIEW_REJECT_FAILURE,
-} from './reviewActions';
+import { REVIEW_APPROVE_FAILURE, REVIEW_REJECT_FAILURE } from './reviewActions';
```

Comment:
Should we keep those ?

Reply:
I think we should try to remove this saga altogether. What prevents you from it?